tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "baseUrl": "src",
+    "noUnusedParameters": true,
```

Comment:
Nit: 

Would this better belong in the .eslintrc to fail the lint, instead of fail the ts build? I've mostly seen it as a linting rule.

Reply:
I'd say lint; there are occasional reasons to have unused parameters, and I'd say overriding a lint rule would be better than overriding a compiler one.

Reply:
Yeah, I'd say it's not a blocker. I tested locally and it's only a warning. My worry was during development it would fail to compile while adding a param or variable that was only brought in mid-devleopment.

Reply:
The `no-unused-variable` TSLint rule requires [type information](https://palantir.github.io/tslint/usage/type-checking/).

This would require extra work to implement, which is fine, except TSLint rules requiring type information aren't as well supported by IDEs (I think VSCode in particular requires additional configuration, or at least it used to).